### PR TITLE
Added content

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
...
+An example script for creating a logging session can be found `here <https://whylogs.readthedocs.io/en/latest/auto_examples/log_dataframe.html#script>`__.
 
 Create a Logger
 ########################
 
-TBD
+Loggers log statistical information about your data. They have the following parameters:
+
+- **dataset_name** sets the name of the dataset, to be used in DatasetProfile metadata and generated filenames.
+- **dataset_timestamp** sets start and end timestamps for the data.
```
Comment (andyndang) [inline docs/getting_started.rst]: this just means "the timestamp" that the data represents (ether the time of the batch or the current data window)

File: docs/getting_started.rst [hunk 2]
```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
...
+- **session_timestamp** sets start and end timestamps for the session.
```
Comment (andyndang) [inline docs/getting_started.rst]: Just means when the session was created

File: docs/getting_started.rst [hunk 3]
```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
...
+
+For more information, see the `documentation <https://whylogs.readthedocs.io/en/latest/autoapi/whylogs/app/logger/index.html>`_ for the logger class.
+
+`This example code <https://whylogs.readthedocs.io/en/latest/auto_examples/configure_logger.html>`_ uses logger options to control the output location. 
 
 Configure a Writer
 ########################
-TBD
+
+Writers write the statistics gathered by the logger into an output file. They have the following parameters:
```
Comment (andyndang) [inline docs/getting_started.rst]: They -> file name template accept the following parameters

File: docs/getting_started.rst [hunk 4]
```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
...
+- **output_path** sets the location output files will be stored. Use a directory path if your writer ``type = 'local'``, or a key prefix for ``type = 's3'``.
+- **formats** lists all supported output formats.
+- **path_template** optionally sets an output path using Python string templates.
+- **filename_template** optionally sets output filenames using Python string templates.
```
Comment (andyndang) [inline docs/getting_started.rst:85]: add dataset_timestamp and session_timestamp here

File: docs/getting_started.rst [hunk 5]
```diff
@@ -37,17 +57,40 @@ Using WhyLogs API
...
+For more information, see the `documentation <https://whylogs.readthedocs.io/en/latest/autoapi/whylogs/app/logger/index.html>`_ for the logger class.
+
+`This example code <https://whylogs.readthedocs.io/en/latest/auto_examples/configure_logger.html>`_ uses logger options to control the output location. 
 
 Configure a Writer
 ########################
-TBD
+
+Writers write the statistics gathered by the logger into an output file. They have the following parameters:
+- **output_path** sets the location output files will be stored. Use a directory path if your writer ``type = 'local'``, or a key prefix for ``type = 's3'``.
```
Comment (bernease) [inline docs/getting_started.rst:82]: Need an empty line between the text and first bullet point for Markdown to format correctly.

**Review by andyndang — APPROVED** [review]

**Pull request merged**